misc/python/materialize/cloudtest/k8s/environmentd.py

```diff
@@ -87,6 +88,8 @@ def generate_stateful_set(self) -> V1StatefulSet:
             field_ref=V1ObjectFieldSelector(field_path="metadata.name")
         )
 
+        coverage = bool(os.getenv("CI_COVERAGE_ENABLED", False))
```

Comment:
I realize there's already precedent for this, but ideally this would be plumbed similarly to `release_mode`, where `CI_RELEASE_MODE` is only read from the environment in conftest.py when pytest is initialized.